app/views/hyrax/collections/show.html.erb

```diff
@@ -55,6 +55,22 @@
 	<div class="row hyc-body">
 		<div class="col-md-8 hyc-description">
 			<%= render 'collection_description', presenter: @presenter %>
+
+			<% if @presenter.collection_type_is_nestable? && @parent_collection_count > 0 %>
+					<div class="row hyc-blacklight hyc-bl-title">
+						<div class="col-md-12">
+							<h2>
+								<%= t('.parent_collection_header') %> (<%= @parent_collection_count %>)
+							</h2>
+						</div>
+					</div>
+					<div class="row hyc-blacklight hyc-bl-results">
```

Comment:
Same as above here.  Could remove the `row` class and the `<div class="col-md-12"></div>` tags.

Reply:
done